requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy>=1.15
+six>=1.11.0
+setuptools>=38.4.0
+scipy>=1.0.1
+PyYAML>=5.1
+mpi4py>=3.0.0
+astropy>=2.0.6
```

Comment:
pyuvdata should be on here too. This list should exactly match what's in the setup.py install-requires. There should not be differences between the versions listed here and there.

Reply:
Removing `pyuvdata` from here made it possible to install using `pip install -r requirements.txt`. Keeping it in, the installation fails because it can't find numpy.

This is to make it possible to install in a completely bare-bones environment with only the standard libraries. This is likely overkill... and I'm also still not clear on what we standard practice is for requirements vs. install-requires. There's that distinction between "package" vs. "application" that I don't quite understand.

Reply:
ok. But I still don't understand the version numbers here being different than the ones in setup.py

Reply:
Good point. It should be fixed now.